### ISSUE TITLE
Get rid of tiny button to switch theme?

### ISSUE THREAD

**Issue by volcbs** [QL-Win/QuickLook#564]
Aesthetically it's unpleasant to see such a tiny button covering small part of your photos or such. Or is there workaround to disable this?

**Comment by xupefei** [comment]
Currently no way unless modifying the source code.
I agree that it is not beautiful but I can't find any other place to put the button. The title bar is already crowded enough :(

**Comment by volcbs** [comment]
Ahh what a bummer
Can you just get rid of it instead (lol) for windows 10? since it already detects based on your theme setting for the respected OS. Like, it detects which OS it is being run then it hides the button for win10 while shows the button for non-win10. S̶o̶m̶e̶h̶o̶w̶ ̶t̶h̶i̶s̶ ̶b̶e̶c̶o̶m̶e̶s̶ ̶a̶ ̶w̶i̶n̶1̶0̶-̶c̶e̶n̶t̶r̶i̶c̶ ̶s̶u̶g̶g̶e̶s̶t̶i̶o̶n̶ ̶l̶o̶l̶.̶

**Comment by xupefei** [comment]
> Ahh what a bummer
> Can you just get rid of it instead (lol) for windows 10? since it already detects based on your theme setting for the respected OS. Like, it detects which OS it is being run then it hides the button for win10 while shows the button for non-win10. S̶o̶m̶e̶h̶o̶w̶ ̶t̶h̶i̶s̶ ̶b̶e̶c̶o̶m̶e̶s̶ ̶a̶ ̶w̶i̶n̶1̶0̶-̶c̶e̶n̶t̶r̶i̶c̶ ̶s̶u̶g̶g̶e̶s̶t̶i̶o̶n̶ ̶l̶o̶l̶.̶

I don't think so. That button is for some specific scenarios, e.g. you want to switch to a dark theme when previewing an all-white image and the otherwise.

I think a better option would be auto-hide. The icon is hidden by default unless you move your mouse to somewhere near it.
